### PR TITLE
Copy spec 1.8 to 1.9

### PR DESCRIPTION
**Pull request by aaronchongth**
# 🎉 New feature

Copying spec 1.8 to 1.9.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [x] Updated migration guide (as needed)
- [x] Code check passed (In source directory, run `sh tools/code_check.sh`)
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review
[another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+)
to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

**Comment by scpeters** [comment]
I've noticed a few places where changes to 1.7 weren't merged forward to 1.8, so I'll open a PR to synchronize those

**Comment by scpeters** [comment]
> I've noticed a few places where changes to 1.7 weren't merged forward to 1.8, so I'll open a PR to synchronize those

see #570 and #571

File: sdf/1.9/root.sdf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<element name="sdf" required="1">
+  <description>SDFormat base element that can include one model, actor, light, or worlds. A user of multiple worlds could run parallel instances of simulation, or offer selection of a world at runtime.</description>
+
+  <attribute name="version" type="string" default="1.8" required="1">
```
Comment (azeey) [inline sdf/1.9/root.sdf:4]: 1.8 -> 1.9

**Review by azeey — APPROVED** [review]
Sorry for the delay. This looks good! Just one minor tweak.

File: sdf/1.9/root.sdf (hunk 1 above)
Comment (aaronchongth) [reply sdf/1.9/root.sdf]: https://github.com/osrf/sdformat/pull/568/commits/f70e9320d8c157ace20815b5d5663fa26778081c

**Pull request merged**